python/spacewalk/satellite_tools/reposync.py

```diff
@@ -382,10 +382,11 @@ def verify_certificates_dates(certs):
         if is_cert:
             cert += line + '\n'
         if is_cert and line.startswith("-----END CERTIFICATE"):
+            is_cert = False
             if not verify_certificate_dates(cert):
-                return False
+                cert = ""
+                continue
```

Comment:
Do I understand this change correctly that we change the date verification from "all certs have valid dates" to "any cert has an valid date"? 

If that's the intention, we could also short-circuit the loop and return `True` as soon as we find the first valid cert.

Reply:
I have to run certificate bundles with this funktion.
And RedHat has some expired certificates in the bundle.
So if 1 cert is provided, it must be valid. If multiple are provided, it is ok if one has a valid date.

Reply:
I change this to return on the first valid certificate